README.rst

```diff
@@ -156,6 +156,48 @@ To change default order use:
     local t = require('luatest')
     t.configure({shuffle = 'group'})
 
+---------------------------------
+Preloaded hooks
+---------------------------------
+Preloaded hooks extend base hooks. They behave like the pytest fixture with the ``autouse`` parameter.
+
+.. code-block:: lua
+    -- my_helper.lua
+    local hooks = require('luatest.hooks')
+
+    hooks.before_all_preloaded(function() print('foo') end)
+
+    hooks.before_all_preloaded(function() print('bar') end)
+    hooks.after_all_preloaded(function() print('baz') end)
+
+    hooks.after_all_preloaded(function() print('qux') end)
```

Comment:
How about the following grouping in the example?

```lua
hooks.before_all_preloaded(function() print('start foo') end)
hooks.after_all_preloaded(function() print('stop foo') end)

hooks.before_all_preloaded(function() print('start bar') end)
hooks.after_all_preloaded(function() print('stop bar') end)
```

Two start/stop actions look more intuitive comparing to abstract 'foo', 'bar', 'baz', 'qux', as for me.

Reply:
Fixed.